src/test/resources/logback.xml

```diff
@@ -9,7 +9,7 @@
     </appender>
 
     <!-- Uncomment for logging ALL HTTP request and responses -->
-     <!--<logger name="io.gatling.http" level="TRACE" />-->
+     <logger name="io.gatling.http" level="TRACE" />
```

Comment:
dunno if you want to leave it logging in CI or not.

Reply:
done